crates/ruma-events/src/room/member.rs

```diff
@@ -93,6 +93,13 @@ pub struct RoomMemberEventContent {
     #[cfg_attr(docsrs, doc(cfg(feature = "unstable-pre-spec")))]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub reason: Option<String>,
+
+    /// Arbitrarily chosen `UserId` (MxID) of a local user who can send an invite.
+    #[cfg(feature = "unstable-pre-spec")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "unstable-pre-spec")))]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[ruma_event(skip_redaction)]
+    pub join_authorised_via_users_server: Option<UserId>,
```

Comment:
Ok so `#[ruma_event(skip_redaction)]` is actually wrong since: in room v8 this wasn't true but 9 fixes this. Oh wait I can just do what the aliases event does right.

Reply:
Yeah, this requires a custom redaction impl like with aliases.